src/Management/src/Endpoint/Actuators/CloudFoundry/PermissionsProvider.cs

```diff
@@ -76,7 +76,7 @@ public async Task<SecurityResult> GetPermissionsAsync(string accessToken, Cancel
                 _logger.LogInformation("Cloud Foundry returned status: {HttpStatus} while obtaining permissions from: {PermissionsUri}", response.StatusCode,
                     checkPermissionsUri);
 
-                return response.StatusCode == HttpStatusCode.Forbidden
+                return response.StatusCode is HttpStatusCode.Forbidden or HttpStatusCode.Unauthorized
```

Comment:
I still find it odd that we return `HttpStatusCode.ServiceUnavailable` (which is a retryable error) when _any other_ error occurs. For example, HTTP 400 for an incorrect response body, or even 404. But at least this improves the current situation.

Reply:
Yeah, it's odd, and I either never knew or don't remember the why for some of it, aside from knowledge we were "just making things work" for Apps Manager or IIS/HWC

Reply:
Logged a follow-up https://github.com/SteeltoeOSS/Steeltoe/issues/1446